odxtools/communicationparameter.py

```diff
@@ -116,6 +119,8 @@ def read_communication_param_ref_from_odx(et_element, doc_frags: List[OdxDocFrag
     else:
         value = read_complex_value_from_odx(et_element.find("COMPLEX-VALUE"))
 
+    is_functional = dl_variant_type == "FUNCTIONAL-GROUP"
```

Comment:
brackets make this more easily readable IMO:
```suggestion
    is_functional = (dl_variant_type == "FUNCTIONAL-GROUP")
```

Reply:
Done